src/server/readpool/config.rs

```diff
@@ -66,4 +98,60 @@ impl Config {
             ..Config::default()
         }
     }
+
+    pub fn validate(&mut self) -> Result<(), Error> {
+        if self.high_concurrency == 0 {
+            return Err(Error::ConfigInvalid(String::from(
+                "readpool.*.high-concurrency should not be 0.",
```

Comment:
Could we define a tag in `ReadPool` to mark it's used in coprocessor or storage?